### DISCUSSION TITLE
Maximum number of locations in the mount

### DISCUSSION THREAD

**Discussion by BinsonBuzz** [trapexit/mergerfs#1083] [General]
For general questions and support please use [GitHub Discussions](https://github.com/trapexit/mergerfs/discussions) or [Discord](https://discord.gg/MpAr69V).

Hi,

I've recently increased the number of locations in my mergerfs mount from about 6 to 11.  Is this too many?  I've started having stability issues (mount dropping occasionally), so I'm wondering if this is the issue?

Regards,

Binson

**Comment by trapexit** [comment]
https://github.com/trapexit/mergerfs/wiki/Real-world-deployments

No. I have over a dozen. People have setups with over 30. There is no limit outside your system's ability to mange it.

Have you looked at your kernels logs to see if it is being killed due to out of memory conditions?
